typed-racket-lib/typed-racket/rep/free-variance.rkt

```diff
@@ -102,6 +102,12 @@
      (remove-frees (flip-variances inner) name)]))
 
 
+(define (make-covariant frees)
+  (combined-frees
```

Comment:
The one use is [here](https://github.com/racket/typed-racket/pull/575/files#diff-78b0804be48e78b95cf9c2b2327596c0R437).

I don't know how to change it.